src/listeners/messages/nonPrefixedMessage.ts

```diff
@@ -1,24 +1,20 @@
 import type { Message } from 'discord.js';
-import { PermissionFlagsBits } from 'discord-api-types/v9';
 import { Events, Listener } from '@sapphire/framework';
+import { isDMChannel, isTextChannel } from '@sapphire/discord.js-utilities';
 
 // The `nonPrefixedMessage` listener is used instead of the base `messageCreate` because, as it was made for message
 // commands, all permission and other related checks that would have to be done anyways would already be done.
 export class UserListener extends Listener<typeof Events.NonPrefixedMessage> {
 	public async run(message: Message) {
-		// We don't specify the `DirectMessages` intent, so it isn't currently possible for the message to be from a DM,
-		// but typescript needs reassurance.
-		if (!message.guild) {
+		if (!isDMChannel(message.channel) || isTextChannel(message.channel)) {
```

Comment:
Currently this will return if the channel isn't a dm channel... Did you mean this:?
```suggestion
		if (!isDMChannel(message.channel) && !isTextChannel(message.channel)) {
```

Reply:
Yes...I have no idea wtf I did there..